app/components/ui/images.tsx

```diff
@@ -11,5 +11,18 @@ type ImageProps = ImgHTMLAttributes<HTMLImageElement> &
   };
 
 export function Image({ ...props }: ImageProps) {
-  return <UnpicImage {...props} />;
+  const [loaded, setLoaded] = useState(false);
+
+  return (
+    <div>
+      <div className={`w-full h-full rounded-3xl ${loaded ? 'bg-transparent' : 'bg-slate-200 animate-pulse'}`}>
```

Comment:
Do we need `bg-transparent`? What's the purpose? Could it just be no background color instead?

```suggestion
      <div className={`w-full h-full rounded-3xl ${loaded ? '' : 'bg-slate-200 animate-pulse'}`}>
```

Reply:
I agree with Andre. I think a square shape "skeleton" while loading isn't consistent when the final image is circular. Nicely, implemented though!